config/checkstyle-examples-checks.xml

```diff
@@ -82,12 +82,11 @@
   </module>
   <module name="RegexpSingleline">
     <property name="id" value="ensureSpaceAfterSpecificSingleLineComment"/>
-    <property name="format" value="^\s*//(ok|violation)[^ ]"/>
-    <property name="message" value="Comments like '//ok' or '//violation' should have a space after //."/>
+    <property name="format" value="^//(ok|violation)"/>
```

Comment:
Please squash this change to first commit.
Test commit should be only input changes